packages/replay-next/components/lexical/plugins/code-completion/findMatches.ts

```diff
@@ -44,16 +51,20 @@ function fetchQueryData(
   replayClient: ReplayClientInterface,
   queryScope: string | null,
   frameId: FrameId | null,
-  pauseId: PauseId | null
+  pauseId: PauseId | null,
+  useOriginalVariables: boolean
 ): {
   properties: WeightedProperty[] | null;
   scopes: Scope[] | null;
 } {
   let properties: WeightedProperty[] | null = null;
 
-  let scopes = null;
+  let generatedScopes: Scope[] | null = null;
```

Comment:
I'm curious– why do we use _generated_ scopes for global/Window properties?

Reply:
I noticed that the original scopes that we get from the backend don't contain a global scope(!).

Reply:
Interesting! I guess that's why we were using generated scopes for this before. I just hadn't thought about it.